test/CuratedModule.t.sol

```diff
@@ -0,0 +1,333 @@
+import { INOAddresses } from "../src/lib/NOAddresses.sol";
+
+import { CuratedModule } from "../src/CuratedModule.sol";
+
+import { Utilities } from "./helpers/Utilities.sol";
+import { Fixtures } from "./helpers/Fixtures.sol";
+
+import { CSParametersRegistryMock } from "./helpers/mocks/CSParametersRegistryMock.sol";
+import { ExitPenaltiesMock } from "./helpers/mocks/ExitPenaltiesMock.sol";
+import { CSAccountingMock } from "./helpers/mocks/CSAccountingMock.sol";
+import { LidoLocatorMock } from "./helpers/mocks/LidoLocatorMock.sol";
+import { WstETHMock } from "./helpers/mocks/WstETHMock.sol";
+import { StETHMock } from "./helpers/mocks/StETHMock.sol";
+import { LidoMock } from "./helpers/mocks/LidoMock.sol";
+import { Stub } from "./helpers/mocks/Stub.sol";
+
+contract CuratedModuleTest is Fixtures, Utilities {
+    uint256 public constant BOND_SIZE = 2 ether;
+
+    LidoLocatorMock public locator;
+    WstETHMock public wstETH;
+    LidoMock public stETH;
+    CuratedModule public cm;
+    CSAccountingMock public accounting;
+    Stub public feeDistributor;
+    CSParametersRegistryMock public parametersRegistry;
+    ExitPenaltiesMock public exitPenalties;
+
+    address internal admin;
+    address internal stranger;
+    address internal nodeOperator;
+
+    function setUp() public virtual {
+        nodeOperator = nextAddress("NODE_OPERATOR");
+        stranger = nextAddress("STRANGER");
+        admin = nextAddress("ADMIN");
+
+        (locator, wstETH, stETH, , ) = initLido();
+
+        feeDistributor = new Stub();
+        parametersRegistry = new CSParametersRegistryMock();
+        exitPenalties = new ExitPenaltiesMock();
+
+        ICSBondCurve.BondCurveIntervalInput[]
+            memory curve = new ICSBondCurve.BondCurveIntervalInput[](1);
+        curve[0] = ICSBondCurve.BondCurveIntervalInput({
+            minKeysCount: 1,
+            trend: BOND_SIZE
+        });
+        accounting = new CSAccountingMock(
+            BOND_SIZE,
+            address(wstETH),
+            address(stETH)
+        );
+        accounting.setFeeDistributor(address(feeDistributor));
+
+        cm = new CuratedModule({
+            moduleType: "curated-module-v2",
+            lidoLocator: address(locator),
+            parametersRegistry: address(parametersRegistry),
+            accounting_: address(accounting),
+            exitPenalties: address(exitPenalties)
+        });
+
+        accounting.setModule(cm);
+
+        _enableInitializers(address(cm));
+        cm.initialize({ admin: admin });
+
+        vm.startPrank(admin);
+        cm.grantRole(cm.CREATE_NODE_OPERATOR_ROLE(), address(this));
+        cm.grantRole(cm.RESUME_ROLE(), address(this));
+        cm.grantRole(cm.DEFAULT_ADMIN_ROLE(), address(this));
+        cm.grantRole(cm.STAKING_ROUTER_ROLE(), address(this));
+        vm.stopPrank();
+
+        cm.resume();
+    }
+
+    function test_changeNodeOperatorAddresses_NoExtendedManagerPermissions_SingleOwner()
```

Comment:
`SingleOwner` means "change from zero addresses"?

Reply:
Also this confuses me a little. Above we have the role `OWNER`, but here we say that the owner is the manager and the reward address. I would still consider changing `OWNER` in role name to avoid confusion.